src/Wrapper.js

```diff
@@ -8,12 +8,70 @@ const checkedProps = {
   onMenuToggle: PropTypes.func,
   onSelection: PropTypes.func.isRequired,
   closeOnSelection: PropTypes.bool,
-  tag: PropTypes.string
+  tag: PropTypes.string,
+  focusGroupOptions: PropTypes.shape({
```

Comment:
Let's leave these details out of prop types. Users can look at the docs for focus-group.